setupdocs.md

```diff
@@ -0,0 +1,182 @@
+
+# Car-Bon System: Technical Setup Guide
+
+## Prerequisites
+
+Before beginning the installation, ensure you have:
+
+-   Administrator access to the project repository of github
+-   Windows operating system

```

Comment:
Not necessarily, running this is wsl is the sole reason to make sure this can be run on Linux or windows using wsl.